### PR TITLE
Theme switcher

### PR DESCRIPTION
**Pull request by kseniaSs**
Add theme switcher button, i used icon from codex/icon (but i think it's not good for button)

If you want to correct design, welcome :hugs:

### CODE REVIEW

File: src/presentation/components/header/ThemeButton.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import Button from '@/presentation/components/button/Button.vue';
+
+const { t } = useI18n();
+const mode = useColorMode();
+
+/**
+ * Change theme from dark/light to light/dark
+ */
+function changeTheme() {
+  mode.value = useColorMode().value === 'light' ? 'dark' : 'light';
```
Comment (neSpecc) [inline src/presentation/components/header/ThemeButton.vue:23]: will it be stored somewhere and accepted on the second visit?

**Review by neSpecc — APPROVED** [review]

File: src/presentation/components/header/ThemeButton.vue (hunk 1 above)
Comment (kseniaSs) [reply src/presentation/components/header/ThemeButton.vue:23]: Yep, it's in the local storage

**Review by slaveeks — APPROVED** [review]

File: src/presentation/components/theme/ThemeButton.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <Button
+    class="header__theme"
```
Comment (neSpecc) [inline src/presentation/components/theme/ThemeButton.vue:3]:
```suggestion
```

File: src/presentation/components/theme/ThemeButton.vue [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    :text="t('home.theme')"
```
Comment (neSpecc) [inline src/presentation/components/theme/ThemeButton.vue:4]:
```suggestion
    :text="t('settings.changeTheme')"
```

File: src/presentation/pages/Settings.vue [hunk 4]
```diff
@@ -1,68 +1,12 @@
...
-        type="primary"
-        :icon="IconSave"
-        @click.passive="onClick"
-      />
-    </div>
-  </div>
-  <div v-else-if="isLoading">
-    Loading...
-  </div>
+  <h1>Settings</h1>
```
Comment (neSpecc) [inline src/presentation/pages/Settings.vue:2]: use i18n please

**Review by neSpecc — APPROVED** [review]

**Pull request merged**